app/controllers/skateparks/searches_controller.rb

```diff
@@ -0,0 +1,8 @@
+module Skateparks
+  class SearchesController < ApplicationController
+    def create
+      render partial: 'search_results', locals: {
+        skateparks: Skatepark.search(params[:search].downcase) }
+    end
+  end
+end
```

Comment:
On second thought maybe we should change this to show? I feel like create implies were adding something to the db. What u think brasputin @alookatommorow